core/src/test/scripts/unit_tests_validate_data.py

```diff
@@ -84,14 +84,15 @@ class DataFileTestCase(LogBufferTestCase):
     particular validator class and collect the log records emitted.
     """
 
-    def validate(self, data_filename, validator_class, extra_meta_fields=None, relaxed_mode=False):
+    def validate(self, data_filename, validator_class, extra_meta_fields=None, relaxed_mode=False, strict_mode=False):
```

Comment:
can we give this a specific name, like `strict_maf_checks`?

Reply:
Done :)